### PR TITLE
Settings - Show raw values in tooltip for list settings

### PR DESCRIPTION
**Pull request by PabstMirror**
image

### CODE REVIEW

File: addons/settings/fnc_gui_settingList.sqf [hunk 1]
```diff
@@ -19,6 +19,8 @@ private _lbData = [];
         _tooltip = localize _tooltip;
     };
 
+    _tooltip =  format ["%1 - %2", _x, _tooltip];
```
Comment (Dystopian) [inline addons/settings/fnc_gui_settingList.sqf:22]:
```suggestion
    _tooltip =  format [["%1 - %2", "%1"] select (_tooltip isEqualTo ""), _x, _tooltip];
```
to remove ` - ` when tooltip is empty

Comment (commy2) [reply addons/settings/fnc_gui_settingList.sqf:22]:
```suggestion
    if (_tooltip isEqualTo "") then {
        _tooltip = str _x;
    } else {
        _tooltip = _tooltip + endl + str _x;
    };
```

Comment (Dystopian) [reply addons/settings/fnc_gui_settingList.sqf:22]: commy2's suggestion looks more consistent with setting name tooltip

**Comment by commy2** [comment]
Someone test if newlines work in listbox tooltips.

**Comment by commy2** [comment]
Not sure about this string key being appriate.

**Pull request merged**